package.json

```diff
@@ -39,7 +39,7 @@
     "file-saver": "^1.3.3",
     "focus-trap": "^6.5.0",
     "fuse.js": "^3.3.0",
-    "graphiql": "~1.5.0",
+    "graphiql": "1.5.18",
```

Comment:
have to pin to this version for now since the next version `1.5.19` of `graphiql` bumps up one of its dependency to `graphql-language-service: 5.0.0+`, which breaks how the `graphql-language-service` get loaded in from webpack

need some time to investigate how to properly upgrade them, in a later PR